### ISSUE TITLE
Accounts SignUp/SignIn

### ISSUE THREAD

**Issue by awatson1978** [UCSC-MedBook/MedBook-Telescope3#10]
Before we can begin the MultiActor Collaboration script, we need to make sure that a signIn/SignUp script is working well.

**Comment by awatson1978** [comment]
Fixed with 01736a54d2910c3259d33b60d7d51dbb7edd8e78

**Comment by awatson1978** [comment]
In order to get the signIn/signUp script working, I had to tweak the header navbar a bit.  The original navbar opened menus based on a hover effect, which doesn't work particularly well with automated browser scripts, headless browsers, or mobile devices.  So, I swapped out the navbar for a fairly [generic bootstrap navbar](http://getbootstrap.com/components/#navbar), which opens menus based on a click event.  Navbar header in the `new-ids` branch now looks like this:

![screen shot 2015-05-05 at 1 15 29 pm](https://cloud.githubusercontent.com/assets/675910/7479839/e8d649c6-f329-11e4-93f2-061ca25b311f.png)

![screen shot 2015-05-05 at 1 16 04 pm](https://cloud.githubusercontent.com/assets/675910/7479842/eb92d81e-f329-11e4-8a4c-1e1f1da740da.png)

Some of the functionality may not be entirely wired back up yet, such as the search box.  We'll wire them back up as part of the test-writting process.
